Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -94,11 +113,14 @@ struct SelectTracks {
       if ((status_prong & (1 << 1)) && abs(trackEta) > etamax_3prong) {
         status_prong = status_prong & ~(1 << 1);
       }
+      if ((status_prong & (1 << 2)) && abs(track.eta()) > etamax_bach) {
+        status_prong = status_prong & ~(1 << 2);
+      }
 
       // quality cut
       if (doCutQuality.value && status_prong > 0) { // FIXME to make a more complete selection e.g track.flags() & o2::aod::track::TPCrefit && track.flags() & o2::aod::track::GoldenChi2 &&
         UChar_t clustermap = track.itsClusterMap();
-        if (!(track.tpcNClsFound() >= d_tpcnclsfound.value &&
+        if (!(track.tpcNClsFound() >= d_tpcnclsfound.value && // is this the number of TPC clusters? It should not be used
```

Comment:
If it should not be used, it should be removed. It should be checked.

Reply:
This is in several tasks, I wanted that we all are on the same page. We can discuss this at the next meeting (next week, since tomorrow's is cancelled).